Tasks/Common/utility-common/telemetry.ts

```diff
@@ -55,7 +56,8 @@ export function emitTelemetry(taskSpecificTelemetry: any) {
                 'agent.proxyurl': tl.getVariable("agent.proxyurl")
             };
             let copy = Object.assign(commonTelem, taskSpecificTelemetry);
-            console.log("##vso[telemetry.publish area=Packaging;feature=NuGetCommand]%s",
+            console.log("##vso[telemetry.publish area=Packaging;feature=%s]%s",
```

Comment:
Is this something other tasks could use? If so, we should probably also send the area as a parameter so we don't get all the tasks' telemetry.

Reply:
Good catch.  I will fix.